client/wfnews-war/src/main/angular/src/app/components/saved/confirmation-dialog/confirmation-dialog.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { CapacitorService } from '@app/services/capacitor-service';
+
+
+@Component({
+  selector: 'saved-confirmation-dialog',
+  templateUrl: './confirmation-dialog.component.html',
+  styleUrls: ['./confirmation-dialog.component.scss']
+})
+export class ConfirmatinoDialogComponent {
```

Comment:
Just a small typo here **ConfirmatinoDialogComponent** should be **ConfirmationDialogComponent**

Reply:
thanks!